x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/tabs/alerts/alerts_tab_content.tsx

```diff
@@ -63,7 +69,7 @@ export const AlertsTabContent = () => {
               <LinkToAlertsPage
                 dateRange={searchCriteria.dateRange}
                 data-test-subj="infraHostAlertsTabAlertsShowAllButton"
-                kuery={`${ALERT_RULE_PRODUCER}: ${INFRA_ALERT_FEATURE_ID}`}
+                kuery={`${hostsWithAlertsKuery}`}
```

Comment:
Are we sure we want to filter by this? 

The query's scalability has changed, and instead of showing all alerts across hosts, we're now only displaying alerts for specific hosts. This could limit the visibility we might need.

Any thoughts?

Reply:
well, as the previous query would only show alerts that were created inside infra, now we are filtering by all the hosts that have more than 1 alert. This maintains the alert count we are showing everywhere

Reply:
Got it! 

The query for "hosts having at least one alert" is executed every time the tab is visited, not just when accessed from the Hosts inventory.